### PR TITLE
Consolidating code

### PR DESCRIPTION
**Pull request by evak6979**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/TL-55

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -28,11 +30,43 @@ jobs:
...
+      LBHPACKAGESTOKEN: ${{ secrets.LBHPACKAGESTOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dotnet format
+        run: dotnet tool install dotnet-format --tool-path ./dotnet-format-local/
+      - name: Run formatter check
+        run: ./dotnet-format-local/dotnet-format --check
+
+  # build-and-test:
```
Comment (humulla) [inline .github/workflows/publish.yml]: I don't think this job should be commented out

File: Hackney.Shared.HousingSearch/Domain/Asset/AssetAddress.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    public class AssetAddress
+    {
+        public static AssetAddress Create(string uprn, string addressLine1, string addressLine2,
+            string addressLine3, string addressLine4, string postCode, string postPreamble)
+        {
+            return new AssetAddress(uprn, addressLine1, addressLine2, addressLine3, addressLine4,
+                postCode, postPreamble);
+        }
+
+        public AssetAddress()
```
Comment (humulla) [inline Hackney.Shared.HousingSearch/Domain/Asset/AssetAddress.cs:12]: Do we need this here?

File: Hackney.Shared.HousingSearch/Gateways/Models/Persons/QueryablePerson.cs [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        [Text(Name = "id")]
+        public string Id { get; set; }
+
+        [Text(Name = "title")]
+        public string Title { get; set; }
+
+        [Keyword(Name = "firstname")]
+        public string Firstname { get; set; }
+
+        [Text(Name = "middleName")]
```
Comment (humulla) [inline Hackney.Shared.HousingSearch/Gateways/Models/Persons/QueryablePerson.cs]: The name should be `middlename` to keep it consistent with the others

File: Hackney.Shared.HousingSearch/Domain/Asset/AssetAddress.cs (hunk 2 above)
Comment (notatiyyah) [reply Hackney.Shared.HousingSearch/Domain/Asset/AssetAddress.cs:12]: Not sure, it's in the [API](https://github.com/LBHackney-IT/housing-search-api/blob/master/HousingSearchApi/V1/Domain/Asset/AssetAddress.cs). @evak6979?

**Review by humulla — APPROVED** [review]

**Pull request merged**